README.md

```diff
@@ -89,7 +89,7 @@ Some of the most common built-in matchers are
 - `s.object(fields)`
 - `s.objectWithOnly(fields)`
 - `s.regex(reg)`
-- `s.string()`
+- `s.string(min, max)`
```

Comment:
good point, my bad for not looking at the whole file rather than just the diff
I agree we should be consistent, and perhaps prefer `s.string({min, max})` unless we have a matcher that has too many options that makes that impractical